README.md

```diff
@@ -346,8 +346,24 @@ lookup (for physical measurements and biospecimen donation) as well as a
 Participant Work Queue. Any of the above parameters can be provided as a query parameter to do
 an exact match. Examples:
 
-    GET /ParticipantSummary?hpoId=PITT
-    GET /ParticipantSummary?hpoId=PITT&state=PIIState_MA
+    GET /ParticipantSummary?awardee=PITT
+    GET /ParticipantSummary?awardee=PITT&state=PIIState_MA
+
+    GET /ParticipantSummary?organization=PITT_FOO 
+    GET /ParticipantSummary?site=SITE_HPO
+
+The participant summary API supports filtering and sorting on last modified time
+and returns only the participants for the appropriate awardee to a matching Service Account.
+The priority of filter is...
+* last modified time
+* participant ID (ascending)
+
+    GET /ParticipantSummary?awardee=PITT&last_modified=2018-02-29
```

Comment:
last_modified should be a timestamp -- either 2018-02-29 12:34:56  UTC+0000 -- not a date.

And normal usage of this will be with a "ge" prefix to mean "greater than or equal to", e.g. something like

GET /ParticipantSummary?awardee=PITT&last_modified=ge2018-02-29 12:34:56  UTC+0000 

(with the space escaped).

I think you're also going to want to talk about how to use this with syncing: 

- Call first without passing in anything for last_modified
- On subsequent calls, pass in last_modified=<MAX value of last_modified returned before minus a buffer of lets say 5 minutes>. (Note this means you'll get the same results multiple times if you call in quick succession.)
- Keep syncing until last_modified is within the past few minutes. At that point you'll be more-or-less caught up. Then you can call again every 10 minutes (or whatever sync period they want to use) to get new things.